ops/devnet/environment.go

```diff
@@ -71,7 +71,7 @@ func defaultProps() map[string]any {
 		"starknet-dev": map[string]any{
 			"image": map[string]any{
 				"image":   "shardlabs/starknet-devnet",
-				"version": "v0.3.5",
```

Comment:
I think this actually isn't used, it uses that helm chart template instead \cc @smickovskid